docs/user/get-started.md

```diff
@@ -0,0 +1,339 @@
+# Get Started
+
+For the one who is interested in Vald, this article will show you how to deploy and run Vald on your k8s cluster.
+This article uses Scylla DB as the backend data store for metadata-management and backup-manager.
+Fashion-mnist is used as an example of a dataset.
+
+1. [About](#About)
+    1. [Main Features](#Main-Features)
+    2. [Requirements](#Requirements)
+2. [Starting Vald on k8s cluster](#Starting-Vald-on-k8s-cluster)
+    1. [Deploy](#Deploy)
+    2. [Run](#Run)
+3. [Advanced](#Advanced)
+
+## About
+
+Vald is a distributed highly scalable and fast approximate nearest neighbor dense feature vector search engine.<br>
+Vald can handle large amount of specific objects, such as image, text, video, and etc.<br>
```

Comment:
can Vald handle these types of objects at the same time?

Reply:
not same time, but we can handle large amount of same type objects